UI/Headless/Application.cpp

```diff
@@ -93,10 +93,18 @@ ErrorOr<void> Application::launch_test_fixtures()
 
     // FIXME: Add option to only run specific fixtures from command line by name
     //        And an option to not run any fixtures at all
-    for (auto& fixture : Fixture::all()) {
+    for (auto const& fixture : Fixture::all()) {
         if (auto result = fixture->setup(); result.is_error())
             return result;
+
+        if (fixture->name() == "HttpEchoServer") {
```

Comment:
Should this be done in the setup for the echo server instead?

Reply:
We'd need to provide access to `web_content_options` in the fixture if that architecture sounds fine. Maybe just `Fixture::setup(WebContentOptions&)`?

I also looked into setting it in the method above here `create_platform_options` but that one is called earlier on (before fixtures are setup)

Reply:
Aren't the options a dirty mutable global variable?

Or are they private. Allowing setup to poke at the options doesn't seem like the worst idea. I mostly dislike the cast and name check here in the generic part

Reply:
> Aren't the options a dirty mutable global variable?

This PR changed `LibWebView/Application::web_content_options()` to return a mutable reference instead of a const reference. Before, we only got access to the mutable version through `create_platform_options`, so it kinda is a dirty global but more so now lol